markdown/admin/packcore.html.md.erb

```diff
@@ -0,0 +1,51 @@
+---
+title: HAWQ packcore utility
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## <a id="core_file_definition"></a>Core file
+
+Core file is a disk file that records the image of a process' memory in case the process crashes or terminates abruptly. This image can be later used to debug the state of process at the time when it was terminated. This information can be very useful to debug the cause failure.
+
+## <a id="packcore_definition"></a>Packcore
+
+Packcore utility helps in packing a core file with its context – executable, application and system shared libraries from the current environment. This information can be later unpacked on a different system and can be used for debugging. Packcore takes a core file, extracts the name of the binary which generated the core and executes `ldd` (List Dynamic Dependencies) to get the required information into a single tar archive.
+
+### <a id="using_packcore"></a>Using packcore 
+
+The packcore utility is located under `${GPHOME}/sbin`. Following are the options for packing a core file:
+
+```shell
+$GPHOME/sbin/packcore <core_file>
+
+or 
+
+$GPHOME/sbin/packcore -b|--binary  $GPHOME/bin/postgres <core_file>
+```
+
+Alternatively, you can navigate to `$GPHOME/sbin` and run the following:
+
+```shell
+./packcore <core_file>
+
+or 
+
+./packcore -b|--binary $GPHOME/bin/postgres <core_file>
+```
+Once finished the utility will create a tar file named `packcore-<core_file>.tgz`. This file can be later unpacked on another system and used for debugging.
```

Comment:
When processing is completed, the utility creates a tar file in the format: `packcore-<core_file>.tgz`. Unpack this file on another system to use it for debugging.